src/applications/personalization/profile-2/components/ProfileSideNav.jsx

```diff
@@ -87,26 +85,35 @@ const ProfileSideNav = ({ closeSideNav, isSideNavOpen }) => {
         />
         <h2 className="vads-u-font-size--h4">Profile</h2>
         <ul>
-          {Object.values(childRoutes).map((route, i) => (
-            <li key={i}>
-              <Link
-                activeClassName="is-active"
-                to={route.path}
-                onClick={() => {
-                  closeSideNav(false);
-                }}
-              >
-                {route.name}
-              </Link>
-            </li>
-          ))}
+          {routes.map(route => {
+            // Do not render route if we don't have the loa for it.
+            if (route.requiresLOA3 && !isLOA3) {
+              return null;
+            }
+
+            return (
+              <li key={route.path}>
+                <NavLink
```

Comment:
Changed to `NavLink` since `Link` in `react-router-dom` does not seem to support `activeClassName` (https://stackoverflow.com/questions/40218050/react-router-link-activeclassname-not-working)

Reply:
Have you tried using [SideNav from the design system](https://design.va.gov/components/sidenav) instead of `<Navlink />` from `react-router-dom`?

Reply:
@erikphansen I feel this question would be more related to your work on the the `ProfileSideNav` component?

In this PR I did not implement a sidenav stategy, I merely changed the link tags.

Reply:
Hey @jhonnyoddball, yes I definitely looked at that SideNav component that you pointed out. But for a number of reasons we had to build a custom React component (based on the existing non-react version) that did what we needed.